docs/api-diffs/9.0/rc.1/diff.md

```diff
@@ -0,0 +1,1290 @@
++            public Func<ExecuteCommandContext, Task<ExecuteCommandResult>> ExecuteCommand { get; }
++            public string? IconName { get; }
++            public IconVariant? IconVariant { get; }
++            public bool IsHighlighted { get; }
++            public object? Parameter { get; }
++            public string Type { get; }
++            public Func<UpdateCommandStateContext, ResourceCommandState> UpdateState { get; }
++        }
++        public sealed class ResourceCommandSnapshot : IEquatable<ResourceCommandSnapshot> {
++            public ResourceCommandSnapshot(string Type, ResourceCommandState State, string DisplayName, string DisplayDescription, object Parameter, string ConfirmationMessage, string IconName, IconVariant? IconVariant, bool IsHighlighted);
++            public string ConfirmationMessage { get; set; }
++            public string DisplayDescription { get; set; }
++            public string DisplayName { get; set; }
++            public string IconName { get; set; }
++            public IconVariant? IconVariant { get; set; }
++            public bool IsHighlighted { get; set; }
++            public object Parameter { get; set; }
++            public ResourceCommandState State { get; set; }
++            public string Type { get; set; }
++        }
++        public enum ResourceCommandState {
++            Disabled = 1,
++            Enabled = 0,
++            Hidden = 2,
++        }
+         public static class ResourceExtensions {
++            public static bool TryGetAnnotationsIncludingAncestorsOfType<T>(this IResource resource, [NotNullWhenAttribute(true)] out IEnumerable<T>? result) where T : IResourceAnnotation;
+         }
+         public class ResourceLoggerService {
+-            [AsyncIteratorStateMachineAttribute(typeof(ResourceLoggerService.<WatchAnySubscribersAsync>d__10))]
+-            public IAsyncEnumerable<LogSubscriber> WatchAnySubscribersAsync([EnumeratorCancellationAttribute] CancellationToken cancellationToken = default(CancellationToken));
++            [AsyncIteratorStateMachineAttribute(typeof(ResourceLoggerService.<WatchAnySubscribersAsync>d__15))]
++            public IAsyncEnumerable<LogSubscriber> WatchAnySubscribersAsync([EnumeratorCancellationAttribute] CancellationToken cancellationToken = default(CancellationToken));
+         }
++        [AttributeUsageAttribute(2048, AllowMultiple=false)]
++        public class ResourceNameAttribute : Attribute, IModelNameParameter {
++            public ResourceNameAttribute();
++        }
+         public class ResourceNotificationService {
+-            [ObsoleteAttribute("ResourceNotificationService now requires an IHostApplicationLifetime.\r\nUse the constructor that accepts an ILogger<ResourceNotificationService> and IHostApplicationLifetime.\r\nThis constructor will be removed in the next major version of Aspire.")]
+-            public ResourceNotificationService(ILogger<ResourceNotificationService> logger);
```

Comment:
I know this was obsoleted already, but we should make sure this is called out in the breaking changes notes anyway. cc: @mitchdenny